src/codeu/chat/client/commandline/Chat.java

```diff
@@ -181,6 +185,23 @@ private UserContext findUser(String name) {
       }
     });
 
+    // Added by Priyanka Agarwal according to CodeU Tutorial
+    // Added import statement
+    panel.register("info", new Panel.Command() {
+      @Override
+      public void invoke(Scanner args) {
+        final ServerInfo info = context.getInfo();
+        if (info == null) {
+          // Communicate error to user - the server did not send us a valid
```

Comment:
Thanks for the feedback, I'll make sure to write the rest of my comments like that.